src/otx/core/config/data.py

```diff
@@ -34,6 +43,8 @@ class DataModuleConfig:
     val_subset: SubsetConfig
     test_subset: SubsetConfig
 
+    tile_config: TilerConfig
```

Comment:
You added the tiling configuration to datamodule only. However, I'm feeling that we need to add a tiling flag to model as well. This is because, when we export the model, we should know whether it is tiled model or not, right?

Reply:
Yea, but I meant that we need it at the model exportation. This is because the exported model (OV IR or ONNX) should be able to

1) take a high resolution image input 
2) tile a high resolution image into low resolution tiled images
3) merge predictions from the tiled images

in standalone. In the current way, we do not distinguish whether it is a tiling model or not after the model exportation.

If you expect that it is not solvable in this PR, please pile this on the backlog, would you?

Reply:
@vinnamkim I added a todo to model.py: https://github.com/openvinotoolkit/training_extensions/pull/2764/files#diff-d6eede8717d7452da4455f96f12096f179062a7610b8b7c9502dcecc32d5ff8cR8